apluslms_roman/cli.py

```diff
@@ -197,17 +198,19 @@ def parse_actioncontext(parser):
         chdir(path_join(*dirs))
 
     # load settings
-    allow_missing = args.config is None
-    config = args.config if args.config is not None else RomanSettings.get_config_path()
-
+    project = "project" in args and args.project
+    allow_missing = args.config is None and not project
+    config = (
+        args.config if args.config is not None
+        else getcwd() + "/" + GlobalSettings.name + ".yml" if project
```

Comment:
I'm not sure what this is supposed to do here? Global settings file should be selected with `--config` flag.

Reply:
some old remnants that I didn't properly edit I guess...?